### DISCUSSION TITLE
HDD Speed emulation opportunities

### DISCUSSION THREAD

**Discussion by resle** [86Box/86Box#2586] [General]
I was wondering what opportunities open up with HDD speed emulation having been implemented (a tremendous achievement imho).

HDD noise emulation comes to mind. 
I don't know if they would be amenable to, but I personally know at least two people with a huge collection of vintage hard drives spanning the years 1979 - 1999 or so.

**Comment by elyosh** [comment]
This is something that I would like to **try** to implement. I was actually thinking of this as a next step when working on the HDD speed emulation.
This would require good quality audio samples of the drive seeking (track-to-track, full stroke, and multiple steps in between).
If someone has the ability to make these recordings I could provide a small tool to generate the required seeking.

**Reply by resle** [reply]
> This is something that I would like to **try** to implement. I was actually thinking of this as a next step when working on the HDD speed emulation. This would require good quality audio samples of the drive seeking (track-to-track, full stroke, and multiple steps in between). If someone has the ability to make these recordings I could provide a small tool to generate the required seeking.

Well it turns out that neither of my collector friends are not willing to help, citing (understandably I have to say) that they have the HDDs but no easy ways to power up and run them.

With that being said, I think the tool would enable distributing the workload to the other collectors out there.
